### PR TITLE
voice message popup

### PR DESCRIPTION
**Pull request by AlisaZil**
please don't merge yet.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -240,7 +281,9 @@
                     <img id="closeGotoVoiceMessagePopupBTN" src="icons/voiceMessage/group.png"
                         onclick="closeGotoVoiceMessagePopup()" />
                 </div>
-            </div>
+            </div> -->
```
Comment (NoamGaash) [inline index.html:284]: delete - it's in the git history if you'll ever need it

File: css/voiceMessegePopup.css [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+.listen-to-voice-messege-button:active{
+    background-color: #116d92;
+}
+.button-text{
+    color: white;
+    font-size: 14px;
+    margin: 0;
+    margin-left:15px;
+}
```
Comment (NoamGaash) [inline css/voiceMessegePopup.css:135]: linebreak

**Review by NoamGaash — APPROVED** [review]
Good job!

File: index.html (hunk 1 above)
Comment (AlisaZil) [reply index.html]: added back this code section 
this popup is needed

**Review by bender2001 — COMMENTED** [review]
hey, looks great but there are some features missing like closing popup with click outside which you can do by add window.addEventListener and in addition you dont have blurred background.

**Comment by AlisaZil** [comment]
> hey, looks great but there are some features missing like closing popup with click outside which you can do by add window.addEventListener and in addition you dont have blurred background.

 יש אופציה של סגירה בלחיצה מחוץ לקומפוננטה.
באפיון אין מסך במצב blur יש מצב דהוי

File: css/voiceMessegePopup.css [hunk 3]
```diff
@@ -12,14 +12,39 @@
...
+/* .progress-bar::-moz-range-thumb{
+    -webkit-appearance: none;
+    background-color: #116d92;
+} */
+/* .progress-line{
+    width: 0%;
+    height: 100%;
+    background-color: #52AEE7;
+    transition: 0.3s;
+} */
```
Comment (NoamGaash) [inline css/voiceMessegePopup.css:47]:
```suggestion
```

File: index.html [hunk 4]
```diff
@@ -237,11 +237,14 @@
                 </div>
 
                 <div class="audio-progress-section">
-                    <audio class="audio-file" src="./audio/efroni.mp3"></audio>
+                    <!-- <audio class="audio-file" src="./audio/efroni.mp3"></audio> -->
```
Comment (NoamGaash) [inline index.html:240]:
```suggestion
```

File: index.html [hunk 5]
```diff
@@ -237,11 +237,14 @@
...
                     <p class="time">0:22</p>
-                    <div class="progress-bar">
-                    </div>
-                    <img class="play-icon" onclick="playOrStopVoiceMessege()"
+                    <input class="progress-bar" type="range" step="0.01">
+                    <!-- <div class="progress-bar" 
+                    onclick="getRangeValue()">
+                        <div class="progress-line"></div>
+                    </div> -->
```
Comment (NoamGaash) [inline index.html:246]:
```suggestion
```

File: js/popups.js [hunk 6]
```diff
@@ -1286,11 +1288,51 @@ function closeVoiceMessegePopup(){
...
 
+
+let voiceMessegeFile = new Audio('./audio/efroni.mp3');
+const progressBar = document.querySelector(".progress-bar");
+const playPauseIcon =  document.querySelector(".play-pause-icon");
+
+
+voiceMessegeFile.onloadedmetadata = () =>{
+    progressBar.max = voiceMessegeFile.duration;
+    progressBar.value = voiceMessegeFile.currentTime;
```
Comment (NoamGaash) [inline js/popups.js:1307]: wouldn't you like to update this value as the file keeps playing?

File: js/popups.js [hunk 7]
```diff
@@ -1286,11 +1288,51 @@ function closeVoiceMessegePopup(){
...
+
+// if(isPlaying){
+//     let progressChnage = setInterval(function(){
+//         progressBar.value = voiceMessegeFile.currentTime;
+//         // console.log(timer++);
+//     }, 100);
+//     if(!isPlaying){
+//         clearInterval(progressChnage);
+//     }
+// }
```
Comment (NoamGaash) [inline js/popups.js]: https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/playing_event

**Review by NoamGaash — APPROVED** [review]

**Pull request merged**